samples/sample_common/src/sample_utils.cpp

```diff
@@ -858,6 +858,58 @@ void CSmplYUVWriter::Close()
     m_bInited = false;
 }
 
+mfxStatus GetChromaSize(const mfxFrameInfo & pInfo, mfxU32 & ChromaW, mfxU32 & ChromaH)
+{
+    mfxStatus sts = MFX_ERR_NONE;
```

Comment:
you don't need this variable, since you just return ```ERR_NONE``` at the end

Reply:
Sure. I will change it. Thanks.